lib/libc/string/wcscmp.c

```diff
@@ -49,10 +49,9 @@ __FBSDID("$FreeBSD$");
 int
 wcscmp(const wchar_t *s1, const wchar_t *s2)
 {
-
 	while (*s1 == *s2++)
-		if (*s1++ == '\0')
```

Comment:
Gratuitous (yes, maybe better, but this is not a "possible overflow when comparing wide char types")

Reply:
Not sure why you think this is resolved when you've neither replied nor dropped the change? IMO this change is fine but either it should be in a separate commit or this one should have its message altered to reflect the actual changes.